### ISSUE TITLE
Doesn't Print totals

### ISSUE THREAD

**Issue by lordfolken** [CptFrikadel/igc-loginator#1]
Hey!

Thx for the utility. 

Unfortunatly specifing several files, doesn't seem to print a "totals" row. 
Also can you maybe make an option --onlytotal ? 
Because i tried to add 150 igc files, but just got a cut-off screen of data.

**Comment by CptFrikadel** [comment]
Hi! 

Thanks for showing interest. It's been a while since  I last worked on this, but I'll implement the fixes/changes as they are quite useful.

**Comment by CptFrikadel** [comment]
Fixed!

- The table now prints a total line at the bottom by default. (As it should)
- The `-onlytotal` flag is now also implemented.  It prints the total number of flights and the total duration of those flights. I removed the 'welcome message' so the output should be sed/awk'able.

Do keep in mind that I originally wrote this tool for use on an openvario, so it makes some assumptions on terminal width and such. I.e. the print pilot option doesn't really work well in narrow (portrait oriented) terminal windows. Scrolling using arrow keys and 'j', 'k' should work fine though.
